### ISSUE TITLE
`SPC l TAB` and `TAB` in the layout-micro-state won't switch to layout 1

### ISSUE THREAD

**Issue by joehillen** [syl20bnr/spacemacs#4436]
If there are 3 layouts open, I can `SPC l TAB` between 2 and 3, but not from 2 to 1, or 3 to 1.

#### System Info
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.2
- Spacemacs branch: develop (rev. 29a7eab)
- Distribution: spacemacs
- Layers:
```elisp
(auto-completion better-defaults dockerfile emacs-lisp evil-commentary eyebrowse floobits git github haskell
                 (ibuffer :variables ibuffer-group-buffers-by 'projects)
                 joehillen markdown org puppet python ranger ruby rust
                 (shell :variables shell-default-height 15 shell-default-position 'bottom shell-default-term-shell "/bin/bash" shell-file-name "/bin/bash")
                 spaceline-top spell-checking syntax-checking themes-megapack vagrant version-control)
```

**Comment by TheBB** [comment]
Fixed by #4444.

**Comment by syl20bnr** [comment]
Should be fixed in 0.105.4(4444...) :-)
